dashboard/app/models/course_offering.rb

```diff
@@ -72,6 +72,15 @@ def self.get_from_cache(key)
     end
   end
 
+  def summarize_for_edit
+    {
+      key: key,
+      is_featured: is_featured?,
```

Comment:
I'm not 100% sure but I thought the pattern here is to use camelCase? So `isFeatured: is_featured?`. I might be wrong though -- this has always been an area I've been confused about.

Reply:
+1 to the confusion. I have never figured out which way we are supposed to go

Reply:
Yeah, definitely not something I'd block this PR on either way

Reply:
It does allow for the cool thing where I can just saving the courseOffering in useState, update it, and then send it right back but I'm guessing I could quickly convert the keys somewhere in that process too

Reply:
`transform_keys(&:underscore)` is useful for this, as well as the reverse `transform_keys(&:camelize)`. But it's still an extra step that doesn't always seem necessary (to me).

Reply:
I'll leave it for now and can update if we decide thats how we want it going forward